src/gleam_json_ffi.mjs

```diff
@@ -0,0 +1,70 @@
+import { Ok, Error } from './gleam.mjs'
+import { bit_string_to_string } from '../../gleam_stdlib/dist/gleam_stdlib.mjs'
+import { UnexpectedByte, UnexpectedEndOfInput } from './gleam/json.mjs'
+
+export function json_to_string(json) {
+  return JSON.stringify(json)
+}
+
+export function object_from(entries) {
+  return Object.fromEntries(entries)
+}
+
+export function array(list) {
+  return list.toArray()
+}
+
+export function do_null() {
+  return null
+}
+
+export function string(x) {
+  return json_to_string(x)
+}
+
+export function int(x) {
+  return parseInt(json_to_string(x), 10)
+}
+
+export function float(x) {
+  return parseFloat(json_to_string(x), 10)
+}
+
+export function bool(x) {
+  return Boolean(x)
+}
+
+
+export function decode(bit_string) {
+  const stringResult = bit_string_to_string(bit_string)
```

Comment:
This conversion to a bit string in JS is slow! Let's always operate on the string when on the JS target, to avoid this conversion. It'll have to be the other way round from Erlang